src/pystatis/config.py

```diff
@@ -32,6 +32,34 @@
     "zensus": re.compile(r"^\d{4}[A-Z]-\d{4}$"),
     "regio": re.compile(r"^((\d{5}-.{1,2}($|-.*$))|(A.*$)|([0-9A-Z]{10}$))"),
 }
+COLUMN_NAME_DICT = {
+    "genesis": {
+        "de": {
+            "time_label": "Zeit_Label",
+            "time": "Zeit",
+            "variable_code": "Auspraegung_Code",
+            "variable_label": "Merkmal_Label",
+            "variable_level": "Auspraegung_Label",
+        },
+        "en": {
+            "time_label": "time_label",
+            "time": "time",
+            "variable_code": "variable_code.1",
+            "variable_label": "variable_label",
+            "variable_level": "variable_code.2",
```

Comment:
is this deliberate that we have `variable_code` here two times, although it is a level? Seems strange to me

Reply:
It seems strange to me too, but that's how genesis and Regionalstatistik name their columns... (at least in the tables I tested)

Reply:
There seems to be an issue with the genesis API itself, the response body maps the columns `Merkmal_Code`, `Auspraegung_Code` and `Auspraegung_Label` all to `variable_code` when setting the language to `en`. Our parsing to a pandas df then creates the .1 and .2 suffixes to avoid three duplicated columns.

Reply:
Contacted API developers. Will incorporate possible changes in future releases.